src/components/Auth/ForgotPassword/ForgotPassword.react.js

```diff
@@ -127,7 +127,7 @@ class ForgotPassword extends Component {
 			<FlatButton
 				label="OK"
 				backgroundColor={
-					UserPreferencesStore.getTheme()==='light' ? '#607D8B' : '#19314B'}
+					UserPreferencesStore.getTheme()==='light' ? '#4285F4' : '#4285F4'}
```

Comment:
@DravitLochan You can let it be `backgroundColor={'#4285F4'}` if you are not using any theme. And delete the UserPreferencesStore. Same for all other places.